modules/nw-osp-configure-octavia-load-balancing-services.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * some post-install one...
+
+[id="nw-osp-configure-octavia-load-balancing-services_{context}"]
+= Enabling {rh-openstack} Octavia for load balancer services
+
+You can create load balancer service types and ingress controllers that have load balancers as backends by using Octavia on {rh-openstack-first}. 
+
+[NOTE]
+====
+Services and controllers that rely on Octavia have the following limitations:
+
+* Only TCP traffic is supported.
+* Active Octavia load balancers and the floating IP addresses that are attached to them are not deleted during a cluster delete operation. They must be deleted beforehand.
+* The `manage-security-groups` property in the cloud provider configuration doesn't not apply to non-administrator tenants.
```

Comment:
`doesn't not` looks like a looks

Reply:
Well spotted. 

```suggestion
* The `manage-security-groups` property in the cloud provider configuration does not apply to non-administrator tenants.
```

Reply:
"administrator tenant" is confusing. I would opt for something like this instead:

```
The `manage-security-groups` property in the cloud provider configuration only applies to OpenStack tenants with administrative privileges.
```

Reply:
```suggestion
* The `manage-security-groups` property in the cloud provider configuration only applies to {rh-openstack} tenants that have administrative privileges. 
```

TY!